plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Main/styles.module.css

```
@@ -79,27 +79,36 @@
       content: unset;
     }
 
-    li,
     .collapsableDiv .anchor svg {
       visibility: hidden;
     }
 
-    li,
+    .collapsableDiv .anchor:focus svg {
+      visibility: visible;
+    }
+
     .collapsableDiv:hover .anchor svg {
       visibility: visible;
     }
 
-    li,
-    .collapsableDiv .anchor:focus svg {
+    .collapsableDiv .anchor {
+      line-height: 2.5;
+    }
+
+    li .anchor svg {
+      visibility: hidden;
+    }
+
+    li .anchor:focus svg {
       visibility: visible;
     }
 
-    li .anchor {
-      line-height: unset;
+    li:hover .anchor svg {
+      visibility: visible;
     }
 
-    .collapsableDiv .anchor {
-      line-height: 2.5;
+    li .anchor {
+      line-height: unset;
```

```suggestion
      line-height: unset;
      margin-left: -17px;
```
reducing the anchor gap should stop flickering.

---

thanks, @yathomasi !